schema/spacewalk/common/data/rhnConfiguration.sql

```diff
@@ -3,3 +3,23 @@ INSERT INTO rhnConfiguration (key, description, default_value) VALUES ('extauth_
 INSERT INTO rhnConfiguration (key, description, default_value) VALUES ('extauth_keep_temproles', 'Keep temporary user roles granted due to the external authentication setup for subsequent logins using password.', 'false');
 INSERT INTO rhnConfiguration (key, description, value, default_value)
 VALUES ('system_checkin_threshold', 'Number of days before reporting a system as inactive', null, 1);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_length_min', 'Minimum number of characters in local user passwords', 4, 4);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_length_max', 'Maximum number of characters in local user passwords', 32, 32);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_lower_char_flag', 'Password has to have at least one lower alpha character', TRUE, TRUE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_upper_char_flag', 'Password has to have at least one upper alpha character', TRUE, TRUE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_digit_flag', 'Password has to have at least one digit', TRUE, TRUE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_consecutive_char_flag', 'Password has to have no consecutive characters', FALSE, FALSE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_special_char_flag', 'Password has to have at least a special character', FALSE, FALSE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_restricted_occurrence_flag', 'Password has to have no repeating characters', FALSE, FALSE);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_max_occurrence', 'Maximum number of valid occurrence of a character', 2, 2);
+INSERT INTO rhnConfiguration (key, description, value, default_value)
+VALUES ('password_check_special_characters', 'List of special characters to check in a password', null, '!$%&()*+,./:;<=>?[\\]^_{|}~');
```

Comment:
We need to think about the defaults. What policy do we want to configure as default?
Maybe something what NIST, BSI & Co suggest?

Reply:
I think we should keep the old defaults, as agreed with @admd, so that we don't break the old tests and the automations already in place

Reply:
Defaulted with previous values for compatibility reason as agreed with @admd